bigframes/session/_io/bigquery/__init__.py

```diff
@@ -222,17 +223,17 @@ def add_labels(job_config, api_name: Optional[str] = None):
 
 
 def start_query_with_client(
```

Comment:
Can we make the `Session` as a class and then `bq_client` and `metrics` can be part of it?

Reply:
Would rather pass more precise context, this makes it more flexible for use in other places, and reduces circular dependency between files.